src/api/helpers/utils.js

```diff
@@ -40,3 +43,62 @@ export function allowOnlyGetMethod(req, res, next) {
 		.status(405)
 		.send({message: `${req.method} method for the "${req.path}" route is not supported. Only GET method is allowed`});
 }
+
+/* eslint-disable*/
+
+export async function getBrowsedRelationships(locals, browsedEntityType, getEntityInfoMethod) {
+	const entity = locals.entity;
+	const {orm} = locals;
+ 	const relationships = locals.relationships;
+	
+	if(! relationships.length > 0) {
+		return [];
+	}
+
+	const filteredRelationships = await relationships
+		.map(relationship => {
+			// Current entity is the source of the relationship
+			if (entity.bbid === relationship.sourceBbid) {				
+				// The other entity is of the browsed type we are looking for
+				// We need a good way to compare entity type strings here and same thing below
+				// Allow for capitalization mistakes? (.toLowercase() on both?)
+				if (relationship.targetEntityType === browsedEntityType) {
+					return {
+						entity: relationship.target,
```

Comment:
How to load entity basic detail here? relationship.target and relationship.source have only bbid and entity type.